elements/pfelement/src/pfelement.js

```diff
@@ -24,9 +24,15 @@ class PFElement extends HTMLElement {
    */
   static debugLog(preference = null) {
     if (preference !== null) {
-      PFElement._debugLog = !!preference;
+      // wrap localStorage references in a try/catch; merely referencing it can
+      // throw errors in some locked down environments
+      try {
+        localStorage.pfeLog = !!preference;
+      } catch (e) {
+        return false;
+      }
     }
-    return PFElement._debugLog;
+    return localStorage.pfeLog === "true";
```

Comment:
Can we persist support for PFElement._debugLog so that the old method continues to work?  We can always flag it for deprecation in 2.0.  We'll also need to update the documentation for how to set this using the new method.

Reply:
Are you supposing that some people may have manually set `PFElement._debugLog = true` instead of using the provided function `PFElement.debugLog(true)`?  If so, we could do:

```
return PFElement._debugLog || (localStorage.pfeLog === "true");
```

Reply:
Yes!  That would be fantastic

Reply:
Cool, added it.  So the fallback doesn't hang around forever, I added a TODO comment to remove it with 2.0.